awapi/autoware_api_msgs/msg/VelocityLimit.msg

```diff
@@ -0,0 +1,2 @@
+builtin_interfaces/Time stamp
+float64 max_velocity
```

Comment:
VelocityLimit aleady exists in autoware_planning_msgs. Why did you add this one?
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/ros2-v0.8.0-beta/common/msgs/autoware_planning_msgs/msg/VelocityLimit.msg